main.md

```diff
@@ -627,11 +627,33 @@ Alternatively, there could be an authorization data type for OpenID Connect. (#o
 
 # Implementation Considerations
 
-## AS Customization
+## Using authorization details in a certain deployment
 
-The scheme and processing will vary significantly among different authorization data types. Any implementation of this draft is therefore supposed to allow the customization of the user consent and the handling of access token data. 
+Using authorization details in a certain deployment will require the follwowing steps:
 
-One option would be to have a mechanism allowing the registration of extension modules, each of them responsible for rendering the respective user consent and any transformation needed to provide the data needed to the resource server by way of structured access tokens or token introspection responses. 
+* Define authorization details types (might include definition and publication of JSON schemas)
+* Publish authorization details types in the OAuth server metadata
+* Determine how authorization details are shown to the user in the user consent 
+* (if needed) Enrich authorization details in the user consent process (e.g. add selected accounts or set expirations)
+* (if needed) Determine how authorization details are reflected in access token content or introspection responses
+* Determine how the resource server(s) process(s) the authorization details or token data derived from authorization details
+
+## Minimal product support
+
+Products supporting this specification should provide the following basic functions:
+
+* Support advertisement of supported authorization details types in OAuth server metadata
+* Accept `authorization_details` parameter in authorization requests including basic syntax check for compliance with this specification 
+* Support storage of consented authorization_details as part of a grant
+* Implement default behavior for carrying authorization details to resource servers access token content and introspection responses. If the product supports resource indicators, this should also include assignment of authorization details to access tokens based on `resource` token request parameter
+
+Processing and presentation of authorization details will vary significantly among different authorization data types. Products should therefore support customization of the respective behavior. In particular products should 
+  
+* allow deployments to determine presentation of the authorization_details
+* allow deployments to modify requested authorization_details in the user consent process, e.g. adding fields 
```

Comment:
Could you give an example of one such modification?
Is that about applying default values to the authorization_details before it gets presented to the user?

Reply:
That’s one option and an example of the AS applying a policy.

Another example is values determined by the user: The client might ask for access to account balance and the user picks the accounts the client shall have access to. It’s described in more detail in the spec: https://tools.ietf.org/html/draft-ietf-oauth-rar-03#section-3.4.1.

Reply:
Thanks Torsten. So that is related to the "Enrich authorization details..." bullet.

Reply:
yes

Reply:
Based on recent discussions around the grant management API, I could also imagine the AS to add expiration times and/or identifiers to authorization details objects.